clients/src/test/java/org/oracle/okafka/tests/OkafkaDescribeTopicsById.java

```diff
@@ -0,0 +1,47 @@
+package org.oracle.okafka.tests;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Map;
+
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.DescribeTopicsResult;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.admin.TopicDescription;
+import org.apache.kafka.common.KafkaFuture;
+import org.apache.kafka.common.TopicCollection;
+import org.apache.kafka.common.Uuid;
+import org.junit.Test;
+import org.oracle.okafka.clients.admin.AdminClient;
+
+public class OkafkaDescribeTopicsById {
+	@Test
+	public void AdminTest() {
+        try (Admin admin = AdminClient.create(OkafkaSetup.setup())) {
+        	CreateTopicsResult result = admin.createTopics(Arrays.asList(
+					new NewTopic("TOPIC",5, (short)1)));
+        	Uuid createdTopicId = result.topicId("TOPIC").get();
+        	admin.close();
+        	
+        	Admin admin1 = AdminClient.create(OkafkaSetup.setup());
```

Comment:
why you have to take new admin client ? 
Also admin1 is not closed .

Reply:
yes, i forgot to discuss this with you, actually there was some issue while using the  same admin client instance.